src/index.ts

```diff
@@ -17,15 +18,19 @@ export default function bundleEnv(inputs : NetlifyPluginOptions<{
   const workingDir = cwd()
   let countFile = 0
   function logDebug(message : string) {
-    if (inputs.debug) {
+    if (inputs.debug && inputs.quiet !== true) {
```

Comment:
Any specific reason to check with `!==`? We could probably just use `!inputs.quiet`, right?